### PR TITLE
Documentation fix

### PR DESCRIPTION
**Pull request by tester12124**
My Documentation fix

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # firstProject
 This is a project where I am going to make my first experience as a committer and as a maintainer
 
-If anybody entered this repo for any random reason, just ignore it.
+If anybody entered this repo for any weird reason, just ignore it.
```
Comment (tester12123) [inline README.md]: No change at all, I'd rather leave it as it was before.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # firstProject
 This is a project where I am going to make my first experience as a committer and as a maintainer
 
-If anybody entered this repo for any random reason, just ignore it.
```
Comment (tester12123) [inline README.md]: test

**Review by tester12123 — COMMENTED** [review]
That's a little different than Gerrit...

**Comment by tester12124** [comment]
I will push another change

**Pull request closed without merging**